src/main/java/com/cupid/jikting/common/config/SecurityConfig.java

```diff
@@ -0,0 +1,95 @@
+package com.cupid.jikting.common.config;
+
+import com.cupid.jikting.common.jwt.filter.JwtAuthenticationProcessingFilter;
+import com.cupid.jikting.common.jwt.service.JwtService;
+import com.cupid.jikting.common.login.filter.CustomJsonUsernamePasswordAuthenticationFilter;
+import com.cupid.jikting.common.login.handler.LoginFailureHandler;
+import com.cupid.jikting.common.login.handler.LoginSuccessHandler;
+import com.cupid.jikting.common.login.service.LoginService;
+import com.cupid.jikting.member.repository.MemberRepository;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.HttpMethod;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.ProviderManager;
+import org.springframework.security.authentication.dao.DaoAuthenticationProvider;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.http.SessionCreationPolicy;
+import org.springframework.security.crypto.factory.PasswordEncoderFactories;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.web.SecurityFilterChain;
+import org.springframework.security.web.authentication.logout.LogoutFilter;
+
+@EnableWebSecurity
+@RequiredArgsConstructor
+@Configuration
+public class SecurityConfig {
+
+    private final LoginService loginService;
+    private final JwtService jwtService;
+    private final MemberRepository memberRepository;
+    private final ObjectMapper objectMapper;
+
+    @Bean
+    public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
+        http
+                .formLogin().disable()
+                .httpBasic().disable()
+                .csrf().disable()
+                .headers().frameOptions().disable()
+                .and()
+                .sessionManagement().sessionCreationPolicy(SessionCreationPolicy.STATELESS)
+                .and()
+                .authorizeRequests()
+                .mvcMatchers("/").permitAll()
+                .mvcMatchers(HttpMethod.POST, "/members").permitAll()
+                .anyRequest().authenticated();
+
+        http.addFilterAfter(customJsonUsernamePasswordAuthenticationFilter(), LogoutFilter.class);
+        http.addFilterBefore(jwtAuthenticationProcessingFilter(), CustomJsonUsernamePasswordAuthenticationFilter.class);
+
+        return http.build();
```

Comment:
p1. 코드를 조금 줄일 수 있을 것 같아요!

```suggestion
        return http
                .formLogin().disable()
                .httpBasic().disable()
                .csrf().disable()
                .headers().frameOptions().disable()
                .and()
                .sessionManagement().sessionCreationPolicy(SessionCreationPolicy.STATELESS)
                .and()
                .authorizeRequests()
                .mvcMatchers("/").permitAll()
                .mvcMatchers(HttpMethod.POST, "/members").permitAll()
                .anyRequest().authenticated()
                .and()
                .addFilterAfter(customJsonUsernamePasswordAuthenticationFilter(), LogoutFilter.class)
                .addFilterBefore(jwtAuthenticationProcessingFilter(), CustomJsonUsernamePasswordAuthenticationFilter.class)
                .build();
```